agent/app/agent_capability_windows_test.go

```diff
@@ -234,3 +234,24 @@ func TestSupportedCapabilitiesWindows(t *testing.T) {
 		assert.Equal(t, aws.StringValue(expected.Value), aws.StringValue(capabilities[i].Value))
 	}
 }
+
+func TestAppendGMSACapabilities(t *testing.T) {
```

Comment:
We also need unit test in linux to make sure we don't advertise the capability for gMSA in linux.

Reply:
Done